### ISSUE TITLE
QR-Code Area missing in frontend

### ISSUE THREAD

**Issue by juekbr** [joshp23/YOURLS-IQRCodes#24]
It worked fine up to a certain point and now the QR code area disappeared in the frontend. I believe it was on the right next to quick share.
I can still mass generate QR codes or configure/mange the plugin in via the menu. Dont know how to analyze to give you more input. Any ideas?

<img width="863" alt="" src="https://user-images.githubusercontent.com/25958524/40729292-5c5fd90e-642c-11e8-9dad-bcd30a7c4b89.png">

**Comment by joshp23** [comment]
h'm, you could
1.  look at your web server logs and see if anything looks odd. 
2.  Make sure the js is loading by looking at the source for that page and seeing if the follwoing shows up:
```
<! --------------------------IQRCodes Start-------------------------- >

<script type="text/javascript">var iqrcodes_imagetype="png";</script>
<script src="https://example.com/user/plugins/iqrcodes/assets/md5.min.js" type="text/javascript"></script>
<script src="https://example.com/user/plugins/iqrcodes/assets/iqrcodes.js" type="text/javascript"></script>
<link rel="stylesheet" href="https://example.com/user/plugins/iqrcodes/assets/iqrcodes.css" type="text/css" />
<link rel="stylesheet" href="/css/infos.css" type="text/css" media="screen" />
<script src="/js/infos.js" type="text/javascript"></script>
<! --------------------------IQRCodes END---------------------------- >
```
3. You can also try opening your web console on Firefox or chrome and seeing what shows up when you click the share button on the admin screen.

**Comment by juekbr** [comment]
Thanks for the quick reply. 
1. looks good
2. looks good
3. Surprisingly I get cross site blocked content warnings in the same line as your javascript is loaded.
the only cause I could think of: I am forwarding all the traffic without a target to one common website.

**Comment by joshp23** [comment]
i suggest trying something like [this](https://addons.mozilla.org/en-US/firefox/addon/cors-everywhere/). Sounds like your set up...?

**Comment by juekbr** [comment]
I ve found the issue :).
We had to convert all the incoming URLs to lower case, cause thats how the tool is being used.
Now, what I didnt expect is that this has an influence on the tool itself as well. I have installed this plugin in a directory containing upper case letter, so thats why I got the errors - the directory (lower case version) wasnt found. Possible solutions: recreate the url rewrites or reinstall the plugin appropriately.

I will close this issue. Thanks for your help
